README.md

```diff
@@ -121,7 +126,8 @@ a(); // Object -- so far so good.
 a().getA(); // "a"
 ```
 
-Yes. Got it. In both of these instances, we actually created a brand new object, and then immediately threw it away, because we didn't assign it to anything. Don't worry about that.
+Yes. Got it. In both of these instances, we actually created a brand new object, and then immediately threw it away, 
+because we didn't assign it to anything. Don't worry about that.
```

Comment:
I'm not sure what you are talking about. Do you like horizontal scrolling?

We, and numerous other projects, have the entire .md files paragraphs mane of multiple lines. See the bottom of this file, for example.

Are we bikeshedding here? :)